experimental/generation/generator/src/library/dialogGenerator.ts

```diff
@@ -275,6 +277,8 @@ async function processTemplate(
                     } else if (filename.includes('library-')) {
                         // Put library stuff in its own folder by default
                         filename = `library/${filename}`
+                    } else if (filename.includes('qna')) {
+                        filename = `${scope.locale}/${filename}`
                     }
```

Comment:
I think this is already covered by the filename including the locale right?  Why would we need for just qna and not lu/lg?

Reply:
I don't think we need to do anything special for .qna. Just put it in the flat locale directory.

---
In reply to: [436976226](https://github.com/microsoft/BotBuilder-Samples/pull/2474#discussion_r436976226) [](ancestors = 436976226)